packages/craco/lib/features/test/create-jest-babel-transform.js

```diff
@@ -0,0 +1,38 @@
+const { isArray } = require("../../utils");
+
+function createJestBabelTransform(cracoConfig) {
+    const babelJest = require("babel-jest");
```

Comment:
Could you extract this require out of the function?

Reply:
Done: it's now at the top of the file